features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64SerializerTest.kt

```diff
@@ -654,6 +654,55 @@ internal class Base64SerializerTest {
         verify(mockSecondCallback).onReady()
     }
 
+    @Test
+    fun `M failover to manual bitmap creation W handleBitmap { bitmapDrawable returned empty bytearray }`(
+        @Mock mockCreatedBitmap: Bitmap
+    ) {
+        // Given
+        whenever(mockBitmapDrawable.bitmap).thenReturn(mockBitmap)
+        whenever(mockBitmap.width).thenReturn(fakeBitmapWidth)
+        whenever(mockBitmap.height).thenReturn(fakeBitmapHeight)
+
+        whenever(mockBitmap.isRecycled)
+            .thenReturn(true)
+            .thenReturn(false)
+
+        val emptyByteArray = ByteArray(0)
+        whenever(mockWebPImageCompression.compressBitmap(any()))
+            .thenReturn(emptyByteArray)
+            .thenReturn(fakeByteArray)
+
+        whenever(mockDrawableUtils.createScaledBitmap(any(), anyOrNull()))
+            .thenReturn(mockBitmap)
+            .thenReturn(mockCreatedBitmap)
+
+        whenever(mockBase64Utils.serializeToBase64String(any()))
```

Comment:
probably for input parameters of the functions called it is possible to use `mockBitmap`, `mockBitmapDrawable` where possible instead of just `any`? or other objects we already know.